components/new-contribution-flow/ContributionFlowStepsProgress.js

```diff
@@ -43,6 +42,42 @@ const STEP_LABELS = defineMessages({
   },
 });
 
+const PrettyAmountFromStepDetails = ({ stepDetails, currency, isFreeTier }) => {
+  if (stepDetails.amount) {
+    const totalAmount = stepDetails.amount + (stepDetails.feesOnTop || 0);
+    return <FormattedMoneyAmount interval={stepDetails.interval} currency={currency} amount={totalAmount} />;
+  } else if (stepDetails.amount === 0 && isFreeTier) {
+    return (
+      <strong>
+        <FormattedMessage id="Amount.Free" defaultMessage="Free" />
+      </strong>
+    );
+  } else {
+    return null;
+  }
+};
+
+const StepInfo = ({ step, stepProfile, stepDetails, stepPayment, isFreeTier, currency }) => {
+  if (step.name === STEPS.PROFILE) {
+    return get(stepProfile, 'name', null);
```

Comment:
In order to match the change in my PR, should be:

`details = get(stepProfile, 'name') || get(stepProfile, 'email', null);`

Reply:
Nice, thanks!